library/notation-issuer-validation/template.yaml

```diff
@@ -53,8 +53,8 @@ spec:
         general_violation[{"result": result}] {
           subject_results := remote_data.responses[_]
           subject_result := subject_results[1]
-          notaryv2_results := [res | subject_result.verifierReports[i].name == "notaryv2"; res := subject_result.verifierReports[i]]
-          issuer_results := [res | notaryv2_results[i].extensions.Issuer == input.parameters.issuer; res := notaryv2_results[i]]
+          notation_results := [res | subject_result.verifierReports[i].name == "notation"; res := subject_result.verifierReports[i]]
```

Comment:
general comment: if we update the verifier name then the new policy in the constraint template we publish will not work for older releases. I'm ok with this breaking change since I believe this will make it tough to maintain multiple versions but want to call this out nonetheless. cc: @binbin-li

Reply:
good call, we could call it out as a breaking change in the next release note.

Reply:
as discussed in the last community, this change is not breaking feature change but more like a config/data change. We'd need to do followings to help users mitigate the breaking behavior:
1. Call it out in the next release
2. Add comments to the template or a separate doc specifying how to make it work with `notation` verifier. And actually it's quite simple as users only need to replace `notaryv2` with `notation` in the template.